package.json

```
@@ -33,6 +33,7 @@
     "diacritics": "^1.3.0",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "lrc-kit": "^1.1.1",
```

With this being a library that hasn't been updated much and not used by many others, could we pin this - just incase future changes aren't desirable.
```suggestion
    "lrc-kit": "1.1.1",
```